.github/workflows/push.yml

```diff
@@ -0,0 +1,37 @@
+name: 'Continuous Integration'
+on:
+  push:
+    branches-ignore:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+jobs:
+  test:
+    runs-on: ubuntu-16.04
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2-beta
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: 'npm install, build, and test'
+      run: |
+        npm install -g npm
+        npm install
+        npm run travis
+  release:
+    needs: test
+    if: github.ref == 'refs/heads/master' && github.event_name == 'push'
+    runs-on: ubuntu-16.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install node.js 12.x
+      uses: actions/setup-node@v2-beta
+      with:
+        node-version: 12.x
+    - name: 'npm install, build, and test'
+      run: |
+        npm install -g npm
+        npm install --production
+        curl "https://raw.githubusercontent.com/pelias/ci-tools/master/semantic-release.sh" | bash -
```

Comment:
the syntax of the `curl` command might need some tweaking, I'm not sure about the mixing of YML, cURL, bash and a pipe 🤷‍♂️